examples/bzlmod/hello_world/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ sh_test(
     name = "hello_world_test",
     srcs = ["hello_world_test.sh"],
     args = [
-        "$(rootpath :hello_world)",
+        "$(rlocationpath :hello_world)",
```

Comment:
This seems more complicated - are there benefits to using `rlocationpath` and needing to pull in a runfiles library, rather than just using `rootpath`?

Reply:
I find `rlocationpath` (use of runfiles) to generally be the most consistent and clear thing to do. I don't want to have to think about where the binary is running relative to some other file. Using the runfiles API feels like a better contract but in this case I'm indifferent. I can remove runfiles.

Reply:
I don't have a super strong preference either way, I'm mostly just a fan of not having to copy+paste 10 lines of runfiles setup into shell scripts :)

Reply:
I feel that. I'd rather this not have been a shell script but don't know if there's a strong reason for this. I think a `rust_test` would have been a better usage here. Just something to execute a rust binary.

Reply:
I think a `rust_test` would be fine :)